docs/optimizing.md

```diff
@@ -0,0 +1,131 @@
+# Optimizing Playbooks
+
+By using all of the following optimizations together its estimated that playbook execution time can be reduced by up to 60% compared to default behavior.
+
+## Using the ACI HTTPAPI plugin
+
+The Ansible ACI HTTPAPI plugin instructs Ansible how to interact with an APIC's HTTP based API and execute tasks on the APIC.
+
+### Benefits
+
+- The ACI login credentials and ansible variables can stay in the inventory.
+- Logs in once and executes subsequent tasks without requiring additional logins when using password-based authentication.
+- Automatically refreshes password-based logins if the token expires during the playbook.
+- Assists with overcoming rate limiting on logins.
+- Leverages APIC's high availability by allowing a list of APIC hosts to be defined as a single ansible host.
+
+### Enabling the plugin
+
+The httpapi plugin can be enabled by setting the following variables:
+
+```ini
+ansible_connection=ansible.netcommon.httpapi
+ansible_network_os=cisco.aci.aci
+```
+
+Instead of using `hostname`, `username` & `password` in the playbook, the following variables can be used in the inventory.
+
+```ini
+ansible_user=admin
```

Comment:
Shall we use ansible_user=admin or ansible_user=apicUser (or something similar)?

Reply:
I used `ansible_user=admin` because it was in ansible-aci README. However, `ansible_user=apicUser` sounds good for the optimization document.